### PR TITLE
OpenTelemetry settings

### PR DESCRIPTION
**Pull request by TheAngryByrd**
Corresponding changes related to https://github.com/fsharp/FsAutoComplete/pull/1060, only merge after FSAC changes have been merged.

### CODE REVIEW

File: release/package.json [hunk 1]
```diff
@@ -665,6 +665,25 @@
...
+        "FSharp.notifications.traceNamespaces": {
+          "default": [
+            "BoundModel.TypeCheck",
+            "BackgroundCompiler."
+          ],
+          "description": "The set of System.Diagnostics.Activity names to watch.",
+          "type" : "array",
+          "required": ["FSharp.notifications.trace"]
+        },
+        "FSharp.openTelemetry.enabled": {
```
Comment (baronfel) [inline release/package.json:682]: This setting needs to be true in order for `notifications.trace` to have any effect, right? If so, I'd add that to the description, and then add that that check to the LanguageService CLI arg condition too.

Comment (TheAngryByrd) [reply release/package.json:682]: No, the notifications uses an `ActivityListener` which is independent of export OTel traces.

Comment (baronfel) [reply release/package.json:682]: oh, perfect :+1:

**Comment by baronfel** [comment]
Tested locally, works great :thumbsup:

**Pull request merged**